jobbergate-api/jobbergateapi2/pagination.py

```diff
@@ -2,18 +2,92 @@
 Pagination feature for all endpoints.
 """
 
+from typing import Generic, List, Optional, TypeVar
+
+from pydantic import BaseModel
+from pydantic.generics import GenericModel
+from sqlalchemy import func, select
+
+from jobbergateapi2.storage import database
+
 
 class Pagination:
     """
     Basic pagination class.
     """
 
-    def __init__(self, limit: int = 10, skip: int = 0):
+    def __init__(self, page: Optional[int] = None, per_page: int = 10):
         """
-        Pagination constructor using default limit and skip offset.
+        Pagination constructor.
+
+        Raises an exception if page is negative values.
+        Raises an exception if per_page is less than 1.
+
+        :param page: The start page. If None, no pagination will be applied.
+        :param per_page: The number of items to include per page.
         """
-        self.limit = limit
-        self.skip = skip
+        if page is not None and page < 0:
+            raise ValueError("The page parameter must be greater than or equal to zero")
+        if per_page < 1:
+            raise ValueError("The per_page parameter must be greater than zero")
+
+        self.page = page
+        self.per_page = per_page
 
     def __str__(self):
-        return f"limit: {self.limit}, skip: {self.skip}"
+        return f"page={self.page}, per_page={self.per_page}"
+
+    def to_dict(self):
+        return dict() if self.page is None else dict(page=self.page, per_page=self.per_page)
```

Comment:
`page` -> `start`
`per_page` -> `limit`

That way the pagination gets the same from the admin API

Reply:
It was "limit" and "skip" before, but Rafael requested a change to the naming and we settled on "page" and "per_page"

Reply:
> 

Yep but the `to_dict` determines the output, doesn't?